src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/LogoutController.cs

```diff
@@ -0,0 +1,88 @@
+using System.Security.Claims;
+using Altinn.Platform.Authentication.Configuration;
+using Altinn.Platform.Authentication.Extensions;
+using Altinn.Platform.Authentication.Model;
+using Altinn.Platform.Authentication.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Authentication.Controllers
+{
+    /// <summary>
+    /// Controller responsible for loging out
+    /// </summary>
+    [Route("authentication/api/v1")]
+    [ApiController]
+    public class LogoutController : ControllerBase
+    {
+        private const string OrginalIssClaimName = "orgiss";
+
+        private readonly GeneralSettings _generalSettings;
+   
+        private readonly OidcProviderSettings _oidcProviderSettings;
+
+        /// <summary>
+        /// Defay
+        /// </summary>
+        public LogoutController(
+            ILogger<LogoutController> logger,
+            IOptions<GeneralSettings> generalSettings,
+            IOptions<OidcProviderSettings> oidcProviderSettings,
+            IOidcProvider oidcProvider)
+        {
+            _generalSettings = generalSettings.Value;
+            _oidcProviderSettings = oidcProviderSettings.Value;
+        }
+
+        /// <summary>
+        /// Logs out user
+        /// </summary>
+        [AllowAnonymous]
+        [ProducesResponseType(StatusCodes.Status302Found)]
+        [HttpGet("logout")]
+        public ActionResult Logout()
+        {
+            ClaimsPrincipal principal = HttpContext.User;
+            string orgIss = principal.GetClaim(OrginalIssClaimName);
+            OidcProvider provider = GetOidcProvider(orgIss);
+            if (provider == null)
+            {
+                return Redirect(_generalSettings.SBLLogoutEndpoint);
```

Comment:
Should Response.Cookies.Delete maybe still be executed before redirect to SBLLogoutEndpoint or are these what SBL still need in order to complete logout in SBL?

Reply:
Will SBL then do a callback to FrontchannelLogout?

Reply:
In the future we will set up SBL as an OIDC provider. Then it will maybe call the logout frontchannel URL. For now we delete cookies in SBL to make sure cookies and sessions is deleted as is